agent/eventhandler/handler.go

```diff
@@ -30,27 +30,20 @@ func HandleEngineEvents(taskEngine engine.TaskEngine, client api.ECSClient, save
 	statesaver = saver
 
 	for {
-		taskEvents, containerEvents := taskEngine.TaskEvents()
+		stateChangeEvents := taskEngine.StateChangeEvents()
 
-		for taskEvents != nil && containerEvents != nil {
+		for stateChangeEvents != nil {
 			select {
-			case event, open := <-containerEvents:
-				if !open {
-					containerEvents = nil
-					log.Error("Container events closed")
+			case event, ok := <-stateChangeEvents:
+				if !ok {
+					stateChangeEvents = nil
+					log.Error("stateChangeEvents closed")
```

Comment:
Can you make this log message more user friendly? Something like `"Unable to handle state change event. The events channel is closed"`